### PR TITLE
HackerRank- Testing- leap year function

### PR DESCRIPTION
**Pull request by Naga-Chaitanya-Paladugu**
Please review this HackerRank's leap year detecting function code.

### CODE REVIEW

File: src/hackerrank_practice/function_leap_year.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# The year can be evenly divided by 4, is a leap year, unless:
+# The year can be evenly divided by 100, it is NOT a leap year, unless:
+# The year is also evenly divisible by 400. Then it is a leap year.
+# This means that in the Gregorian calendar, the years 2000 and 2400 are leap years,
+# while 1800, 1900, 2100, 2200, 2300 and 2500 are NOT leap years. Source
+#
+# Task
+#
+# #Given a year, determine whether it is a leap year. If it is a leap year,
+# #return the Boolean True, otherwise return False.
```
Comment (pcauthorn) [inline src/hackerrank_practice/function_leap_year.py:16]: The function needs to return a boolean.

File: src/hackerrank_practice/function_leap_year.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# #Input Format
+#
+# #Read year, the year to test.
+# #Output Format
+#
+# #The function must return a Boolean value (True/False).
+
+def leap_year(year):
+    if year%4==0:
+        if year%100==0:
```
Comment (pcauthorn) [inline src/hackerrank_practice/function_leap_year.py]: How about an or here?

File: src/hackerrank_practice/function_leap_year.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            else:
+                print(False)
+        else:
+            print(True)
+    else:
+        print(False)
+
+if __name__ == '__main__':
+    year = int(input('Enter a year: '))
+    leap_year(year)
```
Comment (pcauthorn) [inline src/hackerrank_practice/function_leap_year.py:39]: Where the formatting?!

**Review by Naga-Chaitanya-Paladugu — COMMENTED** [review]
Here you go! Please re-review again.

**Review by pcauthorn — APPROVED** [review]

**Review by Naga-Chaitanya-Paladugu — COMMENTED** [review]
Hey, you were right. I made the code more concise using logical operators. Please review it now.

**Pull request merged**